### PR TITLE
Delete and merge loosing children of deleted block.

### PR DESCRIPTION
**Pull request by neotyk**
Somehow this assertion for existence of "test block 3" is not always
failing, but it is exactly the reproduction (that it doesn't exists).

### CODE REVIEW

**Review by filipesilva — APPROVED** [review]

File: test/e2e/delete-doesnt-merge.spec.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+testCleanup = async (page:Page) => {
+    await page.click(".node-page > header > button");
+    // Click button:has-text("Delete Page")
+    await page.click('button:has-text("Delete Page")');
+    // Click button:nth-child(6)
+    await page.click('button:nth-child(6)');
+
+};
+
+test('new-test-template', async ({ page }) => {
```
Comment (filipesilva) [inline test/e2e/delete-doesnt-merge.spec.ts:50]: Nit: rename test please

**Review by filipesilva — COMMENTED** [review]
e2e failing tho, it should pass now

File: test/e2e/delete-doesnt-merge.spec.ts (hunk 1 above)
Comment (neotyk) [reply test/e2e/delete-doesnt-merge.spec.ts]: good call, fixed

File: test/e2e/delete-doesnt-merge.spec.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    await Promise.all ([
+        page.click('button:nth-child(6)'),
+        page.waitForNavigation()
+    ]);
+
+    // Click textarea
+    await page.click('textarea');
+
+    // Invoke Athena, Press k with modifiers
+    await page.press('textarea', 'Meta+k');
```
Comment (filipesilva) [inline test/e2e/delete-doesnt-merge.spec.ts:20]: What I see while debugging is that on linux it hangs waiting for Athena to show up:
```
  pw:api => page.press started +1ms
  pw:api waiting for selector "textarea" +0ms
  pw:api   selector resolved to 2 elements. Proceeding with the first one. +3ms
  pw:api   selector resolved to visible <textarea id="editable-uid-6cbc4d7a0" class="textarea is-e…></textarea> +1ms
  pw:api elementHandle.press("Meta+k") +1ms
  pw:api <= page.press succeeded +14ms
  pw:api => page.fill started +0ms
  pw:api waiting for selector "[placeholder="Find or Create Page"]" +3ms
  pw:api   "networkidle" event fired +88ms
  pw:api => electronApplication.close started +26s
  1) [chromium] › delete-doesnt-merge.spec.ts:50:1 › delete-merge-looses-children ==================

    Timeout of 30000ms exceeded.
```

Maybe the `Meta+k` shortcut doesn't work on linux?

Comment (neotyk) [reply test/e2e/delete-doesnt-merge.spec.ts]: `Meta+k` was it

**Pull request merged**